recipes/testmsm/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "testmsm" %}
+{% set version = "3.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/testmsm-{{ version }}.tar.gz
+  sha256: d6aea2a8820953a5d42059cd411da0f133e90abb91c2d26893046d709fbbe965
+
+build:
+  entry_points:
+    - msmb = msmbuilder.scripts.msmb:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - mdtraj
+    - cython
+    - numpy 1.19
+  host:
+    - pip
+    - python 
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - cython
```

Comment:
Probably don't need `cython` as a run time dependency.

Reply:
Thanks a lot! The problem solved! 
However, I found another issue. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=487255&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2764  
May I ask how to solve it?

Reply:
Hi, I added - libgomp  # [linux], and it seems like problem solved. Thanks again!